.ddev/config.yaml

```diff
@@ -0,0 +1,260 @@
+name: drupal-core-development
```

Comment:
I recommend leaving `name:` out of something like this. It just confuses people. I don't really normally recommend checking in a whole config.yaml in a template project like this, but it's OK. If you remove `name:` then it takes the name of the directory it's in.

```suggestion
```